ethcore/sync/Cargo.toml

```diff
@@ -13,7 +13,7 @@ bytes = { package = "parity-bytes", version = "0.1" }
 client-traits = { path = "../client-traits" }
 common-types = { path = "../types" }
 devp2p = { package = "ethcore-network-devp2p", path = "../../util/network-devp2p" }
-enum_primitive = "0.1.1"
+enum-primitive-derive = { git = "https://gitlab.com/cardoe/enum-primitive-derive" }
```

Comment:
This is updated to get rid of `num_traits 0.1.x`? 

Git dependencies should be avoided if possible IMO, any estimate when `0.1.3` will be released?

Reply:
[Issue tracker entry with request for the next release](https://gitlab.com/cardoe/enum-primitive-derive/-/issues/6).